tcf_core/settings/base.py

```diff
@@ -121,20 +123,20 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, "static/")
-# STATIC_URL = env.str('STATIC_URL', default='static/')
+STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
 
 
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
 DATABASES = {
     'default': {
-        'NAME': 'tcf_db',
+        'NAME': env.str('DB_NAME'),
```

Comment:
Just writing this as a reminder: we'll have to update `.env` in the tCF drive once this gets merged to avoid breaking things.